### DISCUSSION TITLE
Increase Deadzone

### DISCUSSION THREAD

**Discussion by pablopsw** [darthcloud/BlueRetro#981] [Ideas]
Is it possible to increase the Deadzone growth margin? I have found some dreamcast games that are extremely sensitive to the slight movement of the axis, playing with the mapping the only thing that managed to solve the problem was the deadzone, but I feel that it still needs a little to be optimal.

Greetings and thanks.

**Comment by darthcloud** [comment]
Which games?

**Reply by pablopsw** [reply]
First of all, thank you very much for responding.

In the game that is most evident is in Sword of the Berserk Guts' Rage, I have done the test with a DS4, DS3 and Xbox One.

**Comment by kirimui** [comment]
I experienced the same problem with that game and another one that I can't remember the name of. But the character was extremely sensitive to the slightest touch of the stick even when moving alone (my controllers do not have drift, in other games they behave normally), but I do remember wanting to play Berserk and having that problem to the point that it was uncomfortable

**Reply by anodyne3** [reply]
Same issue with Deep Fear on Sega Saturn. I tried setting the deadzone to the max but the issue still exists. I tested this with 2 Xbox Series controllers and a Dual Sense 5. I tested every controller's drift and it's definitely worse on the controller with the highest drift (3.5%) but even the lowest still exhibited this behavior.

It might be the game too though. I noticed that if you move the stick 100% in direction A and let it spring back, it tends to hold the input in the opposite value to direction A. If you gently guide it back to centre it doesn't have the issue.
